### PR TITLE
upgrade netty and add the new netty http codec options

### PR DESCRIPTION
**Pull request by borissmidt**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -466,7 +466,7 @@
     <surefire.redirectTestOutputToFile>true</surefire.redirectTestOutputToFile>
     <source.property>1.8</source.property>
     <target.property>1.8</target.property>
-    <netty.version>4.1.60.Final</netty.version>
+    <netty.version>4.1.69.Final</netty.version>
```
Comment (sullis) [inline pom.xml:469]: Netty 4.1.73.Final is the latest release.

**Review by sullis — COMMENTED** [review]
maybe upgrade to Netty 4.1.73.Final

**Comment by borissmidt** [comment]
@sulis can you merge?

**Comment by yymer** [comment]
@sullis would it be possible to merge this?

**Comment by santoshsssgslab** [comment]
@sullis any update on this PR, when will it be merge?

**Comment by borissmidt** [comment]
sadly we are not maintainers of this library and thus have no power to merge this.

**Comment by hyperxpro** [comment]
I will pull this in v3 release.

**Pull request closed without merging**